include/deal.II/fe/fe_nedelec_sz.h

```diff
@@ -31,13 +31,13 @@ DEAL_II_NAMESPACE_OPEN
 
 /**
  * This class represents an implementation of the
- * H<sup>curl</sup>-conforming N�d�lec element described in the PhD
- * thesis of S. Zaglmayr, <b>High Order Finite Element Methods
- * for Electromagnetic Field Computation</b>, Johannes Kepler
- * Univerit�t Linz, 2006.
+ * H<sup>curl</sup>-conforming N&eacute;d&eacute;lec element described in the
+ * PhD thesis of S. Zaglmayr, <b>High Order Finite Element Methods for
+ * Electromagnetic Field Computation</b>, Johannes Kepler Univerit&auml;t Linz,
```

Comment:
while you're there, the word is also missing an 's'.

Reply:
Will do